lib/smart_proxy_container_gateway/container_gateway_api.rb

```diff
@@ -154,7 +154,8 @@ class Api < ::Sinatra::Base
         do_authorize_any
 
         content_type :json
-        { users: User.map(:name) }.to_json
+        #user = database[:user]
+        { users: ContainerGateway.initialize_db[:users].map(:name) }.to_json
```

Comment:
I'd prefer to use the full namespace
```suggestion
        { users: Proxy::ContainerGateway.initialize_db[:users].map(:name) }.to_json
```